libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -149,7 +149,7 @@ void AP_ADSB::perform_threat_detection(void)
 Location AP_ADSB::get_location(const adsb_vehicle_t &vehicle) const
 {
     Location loc {};
-    loc.alt = vehicle.info.altitude * 100;
+    loc.alt = vehicle.info.altitude * 10; // convert mm to cm.
```

Comment:
Comment says it is converting mm to cm but code is multiplying by 10. Should be divide by 10 right?